bridge/cypress/support/pageobjects/UniformPage.ts

```diff
@@ -62,6 +62,17 @@ class UniformPage {
     return this;
   }
 
+  public openDeletionDialog(): this {
+    cy.byTestId('ktb-danger-subscription-button').click();
+    return this;
+  }
+
+  public deleteSubscriptionFromEditPage(): this {
+    this.openDeletionDialog();
+    cy.get('button').contains('I understand the consequences, delete this subscription').click();
```

Comment:
We should give the delete button a uitestid, but we can also leave it as it is for now.